tests/PathTest.php

```diff
@@ -20,6 +20,26 @@
  */
 class PathTest extends \PHPUnit_Framework_TestCase
 {
+    protected $storedEnv = array();
+
+    public function setUp()
+    {
+        $this->storedEnv['HOME']      = getenv('HOME');
```

Comment:
please don't align the `=`. If you do, we need to change all three lines if the length of the longest line changes, which messes up the Git history